components/search/FilterSidebar.js

```diff
@@ -51,7 +51,7 @@ class FilterSidebar extends React.Component {
 
     // Display `${tomorrow}` as the end date for default search
     // to include the measurements of `${today}` as well.
-    const until = moment().add(1, 'day').format('YYYY-MM-DD')
+    const until = moment().utc().add(1, 'day').format('YYYY-MM-DD')
```

Comment:
It's not introduced in this PR, but I maybe we want to fix it now too.

The until filter does not allow setting the date of tomorrow, although it's the default which is selected.

See:
![Screenshot 2020-01-17 at 12 38 25](https://user-images.githubusercontent.com/424620/72609875-4f49d280-3926-11ea-8860-d3c249e1da89.png)

Reply:
I think this might have to do something with the calendar part of the component which allows till today as a maximum date. If you don't mind, maybe can handle this as a separate issue?

Reply:
Sounds good. I filed this issue for it: https://github.com/ooni/explorer/issues/375